clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/ops/CreateAzureServerGroupAtomicOperation.groovy

```diff
@@ -87,8 +87,8 @@ class CreateAzureServerGroupAtomicOperation implements AtomicOperation<Map> {
         description.securityGroup?.name)
 
       AzureServerGroupNameResolver nameResolver = new AzureServerGroupNameResolver(description.accountName, description.region, description.credentials)
-      description.clusterName = description.name
       description.name = nameResolver.resolveNextServerGroupName(description.application, description.stack, description.detail, false)
+      description.clusterName = description.getClusterName()
```

Comment:
do we need the cluster here since it's basically already part of the server group name?